### PR TITLE
Fix killtimer not going when press f1

### PR DESCRIPTION
**Pull request by NikoCat233**
Not sure whether setting force kill timer will influence other stuffs
Need test probably

Also should be getting rid of mod client having killbutton in lobby by return false in canusekillbutton

The fix is dumb, you gonna need to manually add <size=0%>tohe</size>
It works well though

### CODE REVIEW

**Comment by Pietrodjaowjao** [comment]
NOBODY CAN STOP ME

**Comment by Ultradragon005** [comment]
TY KURWA BLYAT

File: Patches/DialogueBoxPatch.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    }
+
+    [HarmonyPatch(nameof(DialogueBox.Hide)), HarmonyPostfix]
+    public static void Hide_Postfix(DialogueBox __instance)
+    {
+        if (__instance.target.text.Contains("<size=0%>tohe</size>"))
+        {
+            PlayerControl.LocalPlayer.ForceKillTimerContinue = false;
+        }
+    }
```
Comment (Tommy-XL) [inline Patches/DialogueBoxPatch.cs:21]: Wait, doesn't this lead to the fact that when closing cooldown will stop?

**Comment by MargaretTheFool** [comment]
> NOBODY CAN STOP ME

> TY KURWA BLYAT

you guys are so helpful :hehe:

**Comment by Ultradragon005** [comment]
> you guys are so helpful <:hehe:1114307136026050650>
Thnak u, I try my very best

File: Patches/DialogueBoxPatch.cs (hunk 1 above)
Comment (NikoCat233) [reply Patches/DialogueBoxPatch.cs:21]: > Wait, doesn't this lead to the fact that when closing cooldown will stop?

image
image

ForceKillTimerContinue is a single bool
The main decision should be done by iskilltimerenabled

**Comment by Pietrodjaowjao** [comment]
<:extra_clueless:1150848790845194320>

**Comment by Pietrodjaowjao** [comment]
<:uh:1210148084143759360>

**Comment by Ultradragon005** [comment]
thsi is all pietro's fault <:kamikaze:1252290922310406196>

**Comment by NikoCat233** [comment]
> thsi is all pietro's fault :: <:kamikaze:1252290922310406196>

I agree lol

**Comment by Pietrodjaowjao** [comment]
<:heisenberg:1149805810344546334><:heisenberg:1149805810344546334><:heisenberg:1149805810344546334><:heisenberg:1149805810344546334><:heisenberg:1149805810344546334><:heisenberg:1149805810344546334><:heisenberg:1149805810344546334><:heisenberg:1149805810344546334><:heisenberg:1149805810344546334><:heisenberg:1149805810344546334>

**Comment by Ultradragon005** [comment]
<:obsidian69:1252292029312925807><:obsidian69:1252292029312925807><:obsidian69:1252292029312925807><:obsidian69:1252292029312925807>
<:obsidian69:1252292029312925807><:portal:1252292046966886520><:portal:1252292046966886520><:portal:1252292046966886520><:obsidian69:1252292029312925807>
<:obsidian69:1252292029312925807><:portal:1252292046966886520><:portal:1252292046966886520><:portal:1252292046966886520><:obsidian69:1252292029312925807>
<:obsidian69:1252292029312925807><:portal:1252292046966886520><:portal:1252292046966886520><:portal:1252292046966886520><:obsidian69:1252292029312925807>
<:obsidian69:1252292029312925807><:obsidian69:1252292029312925807><:obsidian69:1252292029312925807><:obsidian69:1252292029312925807>

**Comment by Ultradragon005** [comment]
damn it cuttoff

**Comment by Pietrodjaowjao** [comment]
<:rickroll~1:1056776702992334911>

**Comment by Ultradragon005** [comment]
bruh to get the emoji code u can either copy trough phone. or on pc in the server the emoji is in start typing `` click in between them, and then press the emoji u are looking for

**Comment by Pietrodjaowjao** [comment]
animated gifs wont work i thINK

**Comment by Ultradragon005** [comment]
<a:EgirlHype:889830764785594368>

**Comment by Ultradragon005** [comment]
it do

**Comment by Pietrodjaowjao** [comment]
<a:rickroll~1:1056776702992334911>

**Comment by Ultradragon005** [comment]
<a:rickroll:1056776702992334911>

**Comment by Ultradragon005** [comment]
maybe it's some wtf when it has multiple names

**Comment by Pietrodjaowjao** [comment]
oh the emoji name doenst matter

**Comment by Pietrodjaowjao** [comment]
<a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334>

**Comment by Ultradragon005** [comment]
version I meant

**Comment by Ultradragon005** [comment]
<a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334><a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334><a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334><a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334> <a:heisenbergg:1149805810344546334>

**Comment by Pietrodjaowjao** [comment]
<a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334> <a:a:1149805810344546334>

**Comment by Ultradragon005** [comment]
this is so dumb lmao

**Comment by Pietrodjaowjao** [comment]
<a::1149805810344546334>

**Comment by Pietrodjaowjao** [comment]
<:a:1252292029312925807><:a:1252292029312925807><:a:1252292029312925807><:a:1252292029312925807>
<:a:1252292029312925807><:a:1252292046966886520><:a:1252292046966886520><:a:1252292046966886520><:a:1252292029312925807>
<:a:1252292029312925807><:a:1252292046966886520><:a:1252292046966886520><:a:1252292046966886520><:a:1252292029312925807>
<:a:1252292029312925807><:a:1252292046966886520><:a:1252292046966886520><:a:1252292046966886520><:a:1252292029312925807>
<:a:1252292029312925807><:a:1252292029312925807><:a:1252292029312925807><:a:1252292029312925807>

**Comment by Ultradragon005** [comment]
damn almsot there

**Pull request merged**